cmd/derper/derper.go

```diff
@@ -226,7 +230,7 @@ func main() {
 		}
 		go func() {
 			port80srv := &http.Server{
-				Addr:         ":80", // the default, but to be explicit
+				Addr:         net.JoinHostPort(listenHost, "80"), // the default, but to be explicit
```

Comment:
this comment is no longer accurate; I'd just delete it now.

Reply:
Thank you for your advice.